### PR TITLE
Add script for tracking info from new contributors

### PR DESCRIPTION
**Pull request by AlenkaF**
This PR adds the scripts to:

- get GitHub data from last month for new contributors
- save it as a list of dictionaries (should be try to use PyArrow structures?)
- unite code between existing code for all issues and prs (check_issues_prs.py) and the code for new contributors

GitHub Issue: https://github.com/thisisnic/arrowdash/issues/2

### CODE REVIEW

File: scripts/Running the Github API Scripts.ipynb
Comment (AlenkaF) [inline scripts/Running the Github API Scripts.ipynb:1]: Removed this as it was a duplicate of `check_issues_prs.py`.

File: scripts/check_issues_prs.py
Comment (AlenkaF) [inline scripts/check_issues_prs.py:1]: Moved the methods that we will use to _issues_prs.py_.

**Comment by AlenkaF** [comment]
@thisisnic I can't merge this - could you check the settings or even just merge? I think this is ok to get in and I will make follow-up prs to clean up the code and add additional things once we know what we need.

**Comment by thisisnic** [comment]
I'll merge now, and will look into why you can't!

**Pull request merged**